frontend/src/APIClients/mutations/TestMutations.ts

```diff
@@ -30,7 +30,7 @@ export const UNARCHIVE_TEST = gql`
   }
 `;
 
-export const SAVE_ASSESSMENT = gql`
+export const CREATE_NEW_ASSESSMENT = gql`
   mutation Create_Test($test: TestRequestDTO!) {
     createTest(test: $test) {
       id
```

Comment:
updated the naming, since this is the same mutation we should use when pressing the **Publish** button on the Create Assessment page